### ISSUE TITLE
Base_URI not working

### ISSUE THREAD

**Issue by acecx** [openai-php/laravel#108]
I am unable to get the base_uri setting to work when configuring it through app/openai.php.

    'base_uri' => env('PROXY_OPENAI_API', 'https://api.openai.com/v1'),

**Comment by iBotPeaches** [comment]
Yes right now only `api_key`, `organization` and `request_timeout` are configurable in the config. This would be an expansion to support that.

I'll close this one for this existing issue - https://github.com/openai-php/laravel/issues/101 which is about expanding the config for things like this - base_uri, etc.
